src/views/layout.html

```diff
@@ -86,6 +87,6 @@
   {% if FEATURE_FLAG_ENABLE_COOKIE_BANNER == "true" %}
     <script src="/assets/javascripts/cookies.js"></script>
   {% endif %}
-  <script src="//{{CDN_HOST}}/javascripts/govuk-frontend/v3.11.0/govuk-frontend-3.11.0.min.js"></script>
+  <script src="//{{CDN_HOST}}/javascripts/govuk-frontend/v5.3.0/govuk-frontend-5.3.0.min.js"></script>
```

Comment:
Based on new origin `/javascripts` should be removed from the path.

Reply:
Cool, I updated the CDN as the 3.11.0 didn't have some of the styles I needed for this component. Thanks for making the new ticket 👍